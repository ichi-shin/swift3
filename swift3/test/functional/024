#!/bin/bash

# PUT Object ACL

. ./common

S3USER=tester

_s3_put /bucket -H 'x-amz-acl: public-read-write'
_s3_put /bucket/object -T /dev/null -H 'x-amz-acl: private'
_s3_put /bucket/object -T /dev/null -H 'x-amz-acl: public-read'

S3USER=tester2
_s3_get /bucket/object -D - | _filter_curl
