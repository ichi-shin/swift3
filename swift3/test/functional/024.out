QA output created by 024
> s3curl --id tester -- -X PUT -H x-amz-acl: public-read-write http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -T /dev/null -H x-amz-acl: private http://SWIFT_HOST/bucket/object... 200
> s3curl --id tester -- -X PUT -T /dev/null -H x-amz-acl: public-read http://SWIFT_HOST/bucket/object... 200
> s3curl --id tester2 -- -X GET -D - http://SWIFT_HOST/bucket/object... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/octet-stream
Date: DATE
ETag: "ETAG"
Last-Modified: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

